web/src/components/merchant/listings/index.tsx

```diff
@@ -35,35 +31,19 @@ const ListingsContainer = styled.div`
 
 const ListingsPage: React.FC = () => {
   const {hash} = useLocation();
-  const {getMerchant} = useMerchantContext();
-  const [listings, setListings] = useState<Listing[]>([]);
-
-  useEffect(() => {
-    const fetchListings = async () => {
-      const merchant = await getMerchant();
-      if (merchant === undefined) {
-        throw new Error(USER_NOT_SIGNED_IN);
-      }
-
-      const listings = await getAllListings({
-        merchantId: merchant.id.toString(),
-      });
-      setListings(listings);
-    };
-    fetchListings();
-  }, [getMerchant]);
+  const listingsType = hash === '#past-listings' ? 'Past' : 'Ongoing';
+  const {listings} = useListingsApi(listingsType.toLowerCase());
```

Comment:
Prefer
```suggestion
  const listingsType: ListingsType = hash === '#past-listings' ? 'past' : 'ongoing';
  const {listings} = useListingsApi(listingsType);
```
And then for the `header` style in `MerchantPage` set `text-transform: capitalize;`

Reply:
@hzxcaryn Thanks! I didn't realise you can do that with text-transform.